src/pad/padPane.ts

```diff
@@ -1,8 +1,10 @@
-import { authn, icons, ns, pad, widgets } from 'solid-ui'
+import { authn, icons, ns, participation, pad, utils, widgets } from 'solid-ui'
```

Comment:
These changes will not be necessary if we exclude the refactor work from https://github.com/solid/solid-ui/pull/206, like I did in https://github.com/solid/solid-ui/pull/211.